bower.json

```diff
@@ -1,16 +1,31 @@
 {
-    "name": "angular-recaptcha",
-    "version": "2.1.1",
-    "keywords": ["angular", "captcha", "recaptcha", "vividcortex", "human", "form", "validation", "signup", "security", "login"],
-    "main": "release/angular-recaptcha.js",
-    "ignore": [
-        "**/.*",
-        "Gruntfile.js",
-        "src",
-        "node_modules",
-        "bower_components"
-    ],
-    "dependencies": {
-        "angular": "1.*"
-    }
+  "name": "angular-recaptcha",
+  "version": "2.1.1",
+  "keywords": [
+    "angular",
+    "captcha",
+    "recaptcha",
+    "vividcortex",
+    "human",
+    "form",
+    "validation",
+    "signup",
+    "security",
+    "login"
+  ],
+  "main": "release/angular-recaptcha.js",
+  "ignore": [
+    "**/.*",
+    "Gruntfile.js",
+    "src",
+    "node_modules",
+    "bower_components"
+  ],
+  "dependencies": {
+    "angular": "1.*"
+  },
+  "devDependencies": {
+    "angular-mocks": "~1.*",
+    "jquery": "~2.1.3"
```

Comment:
It cleans up after the new reCaptcha.  reCaptcha add this element to the page when the user attempts to click the checkbox.

![image](https://cloud.githubusercontent.com/assets/664714/6601411/e9f456c2-c7eb-11e4-8efc-5922ed9b723a.png)